src/components/WalletAddress.js

```diff
@@ -36,7 +36,7 @@ const mapStateToProps = (state) => {
  *
  * @memberof Components
  */
-class WalletAddress extends React.Component {
+export class WalletAddress extends React.Component {
```

Comment:
For testing, it is recommended to `export` the class in isolation, besides the already existing `export default` that also connects the class with the redux state.